client/src/components/CityCarousel.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+const City = styled.img`
+ height: 100px;
+ width: 180px;
+ padding: 0px 20px;
+`;
+
+class CityCarousel extends Component {
+ render() {
+   const settings = {
+     dots: true,
+     className: "center",
+     centerMode: true,
+     infinite: true,
+     centerPadding: "20px",
+     slidesToShow: 1,
+     speed: 500,
+     rows: 2,
+     slidesPerRow: 2
+   };
+
+   return (
+     <div>
+       <Slider {...settings}>
+         <div>
+           <City src={require("../images/barcelona.jpg")} />
```

Comment:
I recommend trying to find a way to generate these dynamically.  An easy way would be creating a JSON mapping of the cities and looping through it to create these elements.  Since there should already be a cities database, maybe just making a call to retrieve all the cities and then generating off that list.